react-charts/area-chart/intro.md

```diff
@@ -1 +1,15 @@
-In this scenario, you will learn how to use PatternFly React area chart components for the first time.
+In this scenario, you will learn how to use PatternFly React area chart components.
+
+You will begin by adding a CSS selector to define the chart size...
+
+<strong>Note:</strong> Katacoda is setting up a new React application for you. You will be ready to code once the server starts and you can see "Welcome to PatternFly" on the lower pane.
+
+
+Because Victory renders responsive containers, the Chart's width and height props do not determine the width and height of the chart in number of pixels, but instead define an aspect ratio for the chart. 
```

Comment:
This sentence is a bit lengthy. Let's try this instead: Victory renders responsive containers, so the chart's width and height props do not determine the width and height of the chart in number of pixels. Instead, they define an aspect ratio for the chart.